source/taggedalgebraic/taggedalgebraic.d

```diff
@@ -781,6 +781,28 @@ private struct DisableOpAttribute {
 	string name;
 }
 
+/// User-defined attribute to enable only safe calls on the given member(s).
+enum safeOnly;
+///
+@safe unittest
+{
+	union Fields
+	{
+		int intval;
+		@safeOnly int *ptr;
+	}
+
+	// only safe operations allowed on pointer field
+	@safe test() {
```

Comment:
Is this intentionally not executed? It looks like the code could just directly be in the unit test itself, which would then hopefully fix the overall coverage loss.

Reply:
Well, not intentionally. It was meant to show in docs that the code can be done in safe mode (not sure if safe unittests show as such). I didn’t even consider that it is why code coverage is failing. Will fix.